de/data/w_new.csv

```diff
@@ -0,0 +1,83 @@
+1.695123521610829992e-02,-3.043902761448467886e-02,-8.826422352156656700e-03,9.364329431796079156e-03,-1.361813475753141656e-02,5.773218158984316263e-03,-2.388783422159790337e-02,3.497864359258058900e-03,-3.220426914223381565e-02,-8.727620293613576330e-03,-2.039181609409394455e-02,4.051665223708587546e-03,-1.732162646873218470e-02,-5.633643646529272464e-02,-1.729644834264618819e-02,-4.181552005245457553e-03,-4.098638595855062100e-02,-0.000000000000000000e+00,-2.533002339975088790e-03,-0.000000000000000000e+00,-2.021454328263296366e-02,-8.424526731093113550e-06,1.621350009991785691e-04,-6.429837830826234512e-04,1.595744132448633640e-02,-8.740681173136634298e-04,-4.553753978748610923e-04,-4.729352363256992230e-03,2.480068943406127727e-02,-9.428281386321180069e-03,-3.259552452767681963e-02,-1.294988565178333169e-02,-7.228631500641032222e-03,-4.945910504586465469e-05,-2.234998153110052960e-05,-6.692328954699700364e-03,3.054222370951571508e-05,-2.590564549661353300e-02,-5.090690438898139332e-03,1.661477496120356556e-01,-3.836194737139577478e-03,-2.261288264394138714e-03,-6.420093911217967381e-03,-1.401098285753452771e-02,1.599430750111775074e-02,-0.000000000000000000e+00,1.126012538182757233e-02,-8.249610345646767387e-03,-1.334220753090187284e-02,1.335996479442158602e-02,-5.650241656650740150e-03,3.481418978940035713e-01,6.480332390868022083e-03,-2.124142320128601102e-02,6.057458068745778107e-03,3.530451345483161252e-02,-7.258685766899199143e-02,-5.091769721654664307e-03,-3.527164428090458215e-03,-4.939175725068890854e-03,-8.358870108172961706e-04,-1.935832167268243531e-02,-9.404054072212156626e-03,-9.414803068058298380e-04,1.933861270874351890e-01,-9.650495667454874610e-03,-1.077935914508844351e-02,-3.332852341848437008e-03,1.649118006406879652e-02,5.180346116936759490e-03,-1.048049490480711110e-02,-1.384562541388906134e-03,-3.047108418680234015e-04,2.561254970840466400e-03,-7.337445945132452935e-03,3.885125171390090747e-05,-3.536713811822204137e-05,-1.024514810160318673e-02,-2.828108783624424527e-01,2.577169868385379044e-03,-2.195700846557744126e-02,-0.000000000000000000e+00,1.680690888162986174e-03
```

Comment:
What is different about this w matrix?

Reply:
This is the matrix after normalizing according to gene expression under control conditions

Reply:
Can you load the existing data files and calculate this in the code? Having duplicate redundant data files makes it difficult to identify the source data for the analysis.